src/screens/conclusion/sources.js

```diff
@@ -0,0 +1,30 @@
+export const sources = [
+  {
+    text: "How many wild animals are there?",
+    href: "https://reducing-suffering.org/how-many-wild-animals-are-there/",
+  },
+  {
+    text: "Velit rhoncus pellentesque feugiat aenean tincidunt. Diam tincidunt aliquet mauris lorem.",
+  },
+  {
+    text: "Egestas scelerisque ac risus. Neque nibh tellus elementum vulputate auctor.",
+  },
+  {
+    text: "Arcu amet, sollicitudin gravida gravida platea.",
+  },
+];
```

Comment:
I'll defer the PR acceptance to Stephan, but I wanted to chime in about this with what's probably way too many words 😅 

I usually tend to stay away from abstractions like this for two main reasons:

1. I don't believe that duplication of HTML/JSX tags is to be avoided, necessarily. The purpose of HTML is to store data, which is also the purpose of JS arrays/objects, JSON, etc. Essentially, this abstraction trades one data format for another. You save some keystrokes, but...
2. In my experience, abstractions like these tend to lose value as time goes on. In this instance, it may be okay, but if any `Source` ever had different requirements for whatever reason, you'd need to add more properties to the object(s), add code to check for those properties, etc. Over time, it could get messy. Again, it might be fine for this instance, but I personally never take the risk.

For way more words, I think [these](https://overreacted.io/goodbye-clean-code/) [articles](https://sandimetz.com/blog/2016/1/20/the-wrong-abstraction) do a really good job of articulating my opinions.

My proposed suggestion would be to unroll the `.map` so it's like:

```
<Source number={1} href='...'>Text as "children" prop</Source>
<Source number={2} href='...'>Text as "children" prop</Source>
<Source number={3} href='...'>Text as "children" prop</Source> 
```

Anyway, that's all! Up to you/Stephan if you want to change it.

Reply:
I was considering using the "children" approach too so it could be more flexible. Thanks for the insight. Let's see how Stephan weighs in :D

Reply:
I agree with Gerard here, this feels like a premature abstraction. In this case specifically, though, I think we are also reinventing the weel. The `Source` implementation looks exactly like an ordered list (`ol`) to me, which already takes care of the number in front and other things related to accessibility.

My proposal would be to remove the Source component entirely and do the following:
```tsx
<ol className="list-decimal">
  <li><a href="…" rel="noreferrer">How many wild animals are there?</a></li>
  <li>…</li>
  <li>…</li>
</ol>
```

Reply:
Sounds good!